src/components/notification-panel/notification-item.js

```diff
@@ -88,29 +101,36 @@ export default function NotificationItemFactory()
     }
 
     render() {
-      const {notification, removeNotification} = this.props;
+      const {notification, removeNotification, isExpanded} = this.props;
```

Comment:
isExpanded is not part of this component Props. In fact the property is part of the state. You can remove it

Reply:
fixed